cionic/segment.py

```diff
@@ -1,19 +1,23 @@
 """
 Module for segmenting npzs
 """
-import zipfile
-import pathlib
+
 import collections
 import io
 import os
-import logging
+import pathlib
+import zipfile
+
 import numpy as np
 from cionic import from_jsonl, to_jsonl, to_nparray
+
 META_STEMS = ['PATCHES', 'devices', 'segments']
 
 
 def parse_metatables(inputs):
-    'Parse metadata tables from inputs pathnames. Return combined metatables mapping.  Each row includes source_stem for which input it came from.'
+    '''Parse metadata tables from inputs pathnames. Return combined metatables mapping.
```

Comment:
nit: I see a couple of different ways that function header comments are formatted. we should standardize on an approach for the future (but don't need to retroactively apply it)